code/modules/research/xenoarchaeology/artifact/effects/unknown_effect_heal.dm

```diff
@@ -17,14 +17,14 @@
 					if(affecting && istype(affecting))
 						affecting.heal_damage(25 * weakness, 25 * weakness)
 				//H:heal_organ_damage(25, 25)
-				H.vessel.add_reagent("blood",5)
+				H.blood_volume += 5
 				H.nutrition += 50 * weakness
 				H.adjustBrainLoss(-25 * weakness)
 				var/radiation = min(H.radiation, 25 * weakness)
 				H.apply_effect(-radiation,IRRADIATE,0)
 				H.bodytemperature = initial(H.bodytemperature)
 				spawn(1)
-					H.fixblood()
+					H.restore_blood()
```

Comment:
not needed; doesn't do the same thing either; get rid of and the `spawn(1)` as well.